config/locales/en.yml

```diff
@@ -20,4 +20,117 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
```

Comment:
Out of interest: is there a way to have the `en` default to whatever the `t` value is rather than duplicating it?